src/pk/rsa/rsa_import_radix.c

```diff
@@ -25,7 +25,19 @@
 
 #ifdef LTC_MRSA
 
-int rsa_import_radix(int radix, char *N, char *e, char *d, char *p, char *q, char *dP, char *dQ, char *qP, rsa_key *key)
+static int _rsa_read_pk_part(void* mpi, ltc_pk_part *p)
```

Comment:
This might be handy as an internal function e.g. `mp_read_pk_part(..)` or something like that (maybe "pk_part" is not self-explanatory here, but I do not have a better idea).